app/pages/amigos/amigos.ts

```diff
@@ -1,19 +1,44 @@
-import {Component} from "@angular/core";
-import {NavController} from "ionic-angular";
+import {Component, OnInit, Type} from "@angular/core";
 import {DoseAmigosToolbar} from "../../dose-amigos-toolbar/dose-amigos-toolbar.component";
+import {DoseAmigosUserService} from "../../dose-amigos-user-service/dose-amigos-user.service";
+import {DoseAmigosUser} from "../../dose-amigos-user/dose-amigos-user";
+import {UserStatusCardComponent} from "../../user-status-card/user-status-card.component";
+import {FeedPage} from "../feed/feed";
 
 @Component(
     {
         templateUrl: "build/pages/amigos/amigos.html",
         directives: [
-            DoseAmigosToolbar
+            DoseAmigosToolbar,
+            UserStatusCardComponent
         ]
     }
 )
-export class AmigosPage {
+export class AmigosPage implements OnInit {
 
     public title: string = "Amigos";
+    public doseAmigosUsers: Array<DoseAmigosUser> = [];
+    public userStatusClickPage: Type;
 
-    constructor(public nav: NavController) {}
+    constructor(
+        private doseAmigosUserService: DoseAmigosUserService
+    ) {
+
+    }
+
+    /**
+     * Initialize the page.
+     * @returns {Promise}
+     */
+    ngOnInit(): any {
+
+        this.userStatusClickPage = FeedPage;
```

Comment:
For now, clicking on an Amigo takes you to the News Feed Page.

We will want this to eventually take you to a Amigo Feed Page (news feed filtered to only a specific amigo's events).